frontend/packages/kubevirt-plugin/src/components/create-vm/dev-console/dev-console-create-vm-form.tsx

```diff
@@ -0,0 +1,218 @@
+                        className="co-catalog-item-icon__img co-catalog-item-icon__img--large"
+                        src={getTemplateOSIcon(template)}
+                      />
+                    </span>
+                    <div>
+                      <h2 className="co-section-heading co-catalog-item-details__name">
+                        {getTemplateName(template)}
+                      </h2>
+                      <ul className="list-inline">
+                        <li className="co-break-work">
+                          <ExternalLink
+                            href={SUPPORT_URL}
+                            text="Learn more about template support"
+                          />
+                        </li>
+                      </ul>
+                    </div>
+                  </div>
+                  <p className="co-catalog-item-details__description">
+                    {isCommonTemplate(template)
+                      ? 'This template is provided by Red Hat. The Boot source is also maintained by Red Hat.'
+                      : `This template is provider by ${getTemplateProvider(template)}`}
+                  </p>
+                  <>
+                    <hr />
+                    <p>The following resources will be created:</p>
+                    <ul>
+                      <li key="virtual-machine">Virtual machine</li>
+                    </ul>
+                  </>
+                </div>
+              </div>
+              <div className="col-md-5 col-md-pull-7">
+                <Stack hasGutter>
+                  <StackItem>
+                    <CreateVMForm
+                      dispatch={dispatch}
+                      state={state}
+                      template={selectedTemplate}
+                      sourceStatus={sourceStatus}
+                      canCustomize={false}
+                      showCreateInfo={false}
+                    />
+                  </StackItem>
+                  <StackItem>
+                    <ButtonBar
+                      inProgress={isSubmitting}
+                      errorMessage={
+                        createError ||
+                        (!isTemplateSourceError(sourceStatus) && sourceStatus?.isCDRom
+                          ? scError
+                          : undefined)
+                      }
+                    >
+                      <ActionGroup className="pf-c-form">
+                        <Button
+                          type="submit"
+                          variant="primary"
+                          isDisabled={
+                            !state.isValid ||
+                            isSubmitting ||
+                            (!isTemplateSourceError(sourceStatus) && sourceStatus?.isCDRom
+                              ? !scLoaded || !!scError
+                              : false)
+                          }
+                          onClick={async () => {
+                            try {
```

Comment:
I think you should delete any error on retry
```suggestion
setCreateError(null);
try {
```